medical_appointment/models/medical_appointment.py

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2016 LasLabs Inc., Komit
```

Comment:
The copyright isn't binding unless it's prefaced with `Copyright`, so I'm not sure how combining under the one would work TBH. This is probably the best way:
```
Copyright 2016 LasLabs Inc.
Copyright 2017 Komit
```

Note that I don't have a copyright in 2017 - I didn't add any code during that year at this point, only you have. That same reasoning is why I said to get rid of my indefinite copyright on the one `2015-TODAY`.